cpapp_backend/cpapp/services/agent.py

```
@@ -2303,6 +2362,61 @@ def process_prefill_data_for_basic_details(self, session_id: str) -> str:
+                                    # Convert from DD-MM-YYYY to YYYY-MM-DD format if needed
+                                    try:
+                                        if '-' in date_of_birth and len(date_of_birth.split('-')) == 3:
+                                            parts = date_of_birth.split('-')
+                                            if len(parts[2]) == 4:  # DD-MM-YYYY format
+                                                data['dateOfBirth'] = f"{parts[2]}-{parts[1]}-{parts[0]}"
+                                            else:  # Already in YYYY-MM-DD or other format
+                                                data['dateOfBirth'] = date_of_birth
+                                        else:
+                                            data['dateOfBirth'] = date_of_birth
+                                        logger.info(f"Extracted date of birth from PAN360: {data['dateOfBirth']}")
+                                    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)